plugins/meta/vrf/README.md

```diff
@@ -0,0 +1,27 @@
+# vrf plugin
+
+## Overview
+
+This plugin creates a [VRF](https://www.kernel.org/doc/Documentation/networking/vrf.txt) in the network namespace and assigns it the interface passed in the arguments. If the VRF is already present in the namespace, it only adds the interface to it.
+
+As a table id is mandatory, the plugin generates a new one for each different VRF that is added to the namespace.
+
+It does not create any network interfaces and therefore does not bring connectivity by itself.
+It is only useful when used in addition to other plugins.
+
+## Configuration
```

Comment:
can you make more detailed README with examples on how to use it with another plugins?
i.e. https://github.com/containernetworking/plugins/tree/master/plugins/meta/firewall

Reply:
Sure, will do!